arbor/backends/multicore/cable_solver.hpp

```diff
@@ -74,71 +66,82 @@ struct cable_solver {
         }
     }
 
-    const_view solution() const {
-        // In this back end the solution is a simple view of the rhs, which
-        // contains the solution after the matrix_solve is performed.
-        return rhs;
-    }
+    // Setup and solve the cable equation
+    // * expects the voltage from its first argument
+    // * will likewise overwrite the first argument with the soluction
+    template<typename T>
+    void solve(T& rhs, const_view dt_intdom, const_view current, const_view conductivity) {
```

Comment:
Do you know what the reasoning was behind the original split into assemble/solve?

Reply:
It's much more useful on the GPU side, where the memory is shuffled into place to accomodate the 
kernel's thread layout. So, I presume it was done for homogeneity.